SofaKernel/modules/SofaCore/src/sofa/core/collision/BroadPhaseDetection.h

```diff
@@ -81,17 +65,8 @@ class BroadPhaseDetection : virtual public Detection
     sofa::helper::vector< CollisionModelPair > cmPairs;
     std::map<Instance,sofa::helper::vector< CollisionModelPair > > storedCmPairs;
 
-    void changeInstanceBP(Instance inst) override
-    {
-        storedCmPairs[instance].swap(cmPairs);
-        cmPairs.swap(storedCmPairs[inst]);
-    }
+    void changeInstanceBP(Instance inst) override;
 };
 
-} // namespace collision
-
-} // namespace core
-
-} // namespace sofa
-
+}
 #endif
```

Comment:
not that cosmetic: we are trying to switch to `pragma once` instead of the `ifdef guards` (while you are cleaning the headers 😅

Reply:
Good!